wordpress-mobile/WordPress-iOS
# Posts & Pages: Extract search to a separate screen (and ViewModel)

Extract search to a separate screen (and ViewModel)

To test:

- Open the new search and verify that it's showing (some) local results (the way it searches will completely change in future versions)

Known issue: search results don't reload after you exit/re-enter. Please ignore it for now. There is actually a bunch of bugs, but the main goal was to unblock the development for the main screen, so it's all good. Next PR reworks the search.

## Regression Notes
1. Potential unintended areas of impact: Posts & Pages
2. What I did to test those areas of impact (or what existing automated tests I relied on): manual
3. What automated tests I added (or what prevented me from doing so): n/a (yet)

PR submission checklist:


UI Changes testing checklist:
